assets/scripts/aircraft.js

```diff
@@ -1805,14 +1805,32 @@ var Aircraft=Fiber.extend(function() {
         // ALTITUDE
 
         var distance = null;
-        var expedite_factor = 1.7;
+        var expedite_factor = 1.5;
         this.trend = 0;
         if(this.target.altitude < this.altitude - 0.02) {
-          distance = -this.model.rate.descent * game_delta() / expedite_factor;
+          distance = -this.model.rate.descent/60 * game_delta();
           if(this.mode == "landing") distance *= 3;
           this.trend -= 1;
         } else if(this.target.altitude > this.altitude + 0.02) {
-          distance =  this.model.rate.ascent  * game_delta() / expedite_factor;
+          function getClimbRate(ac) {
```

Comment:
@tedrek I'm still kind of trying to climb my way up the call stack and figure out where all the updates to each aircraft are actually being told to happen... I was concerned that if I made it a method, this code block would be run every time the position is updated, even in level flight, which would be an unnecessary expense to the computer.